assets/docs/pages/security/cors.md

```diff
@@ -0,0 +1,434 @@
+Enforce client-site access controls with cross-origin resource sharing (CORS).
+
+{{< callout type="warning" >}} 
```

Comment:
we say this is 2.1 or later, I think we should then remove it from the 2.0 security section